src/cascadia/UnitTests_TerminalCore/SelectionTest.cpp

```diff
@@ -614,11 +614,11 @@ namespace TerminalCoreUnitTests
                 //       start            click      finish
                 term.SetSelectionEnd({ 21, 10 }, Terminal::SelectionExpansion::Word);
 
-                // Validate selection area: "doubleClickMe dragThroughHere" selected
-                ValidateLinearSelection(term, { 4, 10 }, { 32, 10 });
+                // Validate selection area: "doubleClickMe dragThroughHere " selected

```

Comment:
i thought this was behavior we didn't want to have - where selecting a word selects the following whitespace

Reply:
or, is the comment wrong? i guess it would be `34` below if we wanted to include the space

Reply:
yup, comment was wrong. Just double checked it.